db/seeds/base/add_applications.rb

```diff
@@ -0,0 +1,26 @@
+school_settings = %w[a_school an_academy_trust a_16_to_19_educational_setting]
+
+Application.create!(
```

Comment:
I wonder if we could retain the original application logic here (see comment about users above) to ensure we create multiple single application user and multiple application users?